packages/pwa-kit-react-sdk/src/ssr/universal/components/redirect-with-status/index.jsx

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright (c) 2024, Salesforce, Inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+import React from 'react'
+import {Redirect, Route} from 'react-router-dom'
+import PropTypes from 'prop-types'
+
+/**
+ * The `RedirectWithStatus` HOC component is used to specify a different status code when redirecting via
```

Comment:
Did you run into some doc's that called called this component an HOC? I don't think this is a hoc